### ISSUE TITLE
Project Status

### ISSUE THREAD

**Issue by jgarte** [idris-community/idris2-nvim#35]
Hi, is this project still active or is there another fork that I should be using of the same project?

Any guidance here is much appreciated.

**Comment by mattpolzin** [comment]
I can’t answer your question directly but I can offer to maintain my fork of it more officially if that’s desirable to @ShinKage. I still use this plugin actively right now.

**Comment by Russoul** [comment]
Shall we merge your fork into the upstream (here)? 
I use LSP almost daily as well

**Comment by Russoul** [comment]
I can help with reviews, reconciliation, cleanup, etc.

**Comment by mattpolzin** [comment]
Yeah, idris-community would be a good place for a fork of this though. For now we can wait and see if @ShinKage has time to get back to us but I can move my fork to the community org (I think).

**Comment by ShinKage** [comment]
I am unable to maintain my projects at this point in time, and I think moving this to the idris-community is a good idea. When possible, I will come back there.
Let me know if you prefer to move this repo or the fork there, in such case I will modify the README to point to the new repo and archive this one.

**Comment by mattpolzin** [comment]
@ShinKage if you can move this repo to the community org that would be ideal I think -- maintain the through line to your work while making it possible for myself and others to accept PRs to it because we are members of the community org.

Let me know if that is something you cannot or don't have time to do and I will instead move my fork and follow up here.

**Comment by mattpolzin** [comment]
I'll close this. The project has been moved to the community org and I know there are at least a couple of community maintainers who use this plugin so the answer to the original question is: yes, it is being maintained now.
